packages/integration-tests/src/EntityManager.clone.test.ts

```diff
@@ -0,0 +1,270 @@
+import { insertAuthor } from "@src/entities/inserts";
+import { EntityConstructor, EntityManager } from "joist-orm";
+import { Author, Book, Comment, Image, ImageType, newAuthor, newBook, newPublisher, Publisher, Tag } from "./entities";
+import { newEntityManager } from "./setupDbTests";
+
+describe("EntityManager", () => {
```

Comment:
FYI - I thought these tests were getting sufficiently long to split out from `EntityManager.test.ts`. New test cases start at line 170

Reply:
TY - I meant to change that 👍 
```suggestion
describe("EntityManager.clone", () => {
```